### ISSUE TITLE
[bug] Discord crashes with Javascript error after install

### ISSUE THREAD

**Issue by Emmasworld** [BetterDiscord/BetterDiscord#856]
**Describe the Bug**
What the title says - Discord crashing with a Javascript error after installing BD.

**To Reproduce**
Install BD, open Discord.

**Expected Behavior**
Discord launches.

**Screenshots**
![Screenshot at 2021-06-29 17-50-30](https://user-images.githubusercontent.com/48417442/123872984-cbaa5880-d903-11eb-8d92-c0d31e667d73.png)

**Discord Version**
uh... whichever's the most current, I guess? I actually don't know how to check... :sweat_smile:

**Additional Context**
Am running Linux Ubuntu MATE 20.04.02 LTS - maybe this is somehow related to my issue?

**Comment by Copimox** [comment]
Same thing but on Windows and DiscordCanary

**Comment by nchristopher** [comment]
This is a known issue, see #821.
> Discord's latest update includes pushing Electron 13, please be patient while fixes are made. In the meantime use stable or just uninstall BD on Canary.

**Comment by rauenzi** [comment]
How did you install this? Do you have the installer log?

**Comment by Copimox** [comment]
Where do I find the log?

**Comment by rauenzi** [comment]
It's on screen when you install

**Comment by Emmasworld** [comment]
> How did you install this? Do you have the installer log?

Okay, so... I didn't actually expect to get a response when I posted this, but I guess I should mention - I fixed the issue a bit after I had posted this.
It had turned out that I somehow managed to install BetterDiscord as root. I legitimately don't even know how I had managed that, but the biggest hint when I figured it out (and thus reinstalled BetterDiscord, which fixed the problem) was that the error starts off with it not finding a module, and the attempted filepath started with '/root/'. (which if you pay attention to the screenshot in my initial post, is right there at the top of the error, haha)
So basically - the issue was entirely on my end, no worries here.
